include/SDL_config_amiga.h

```diff
@@ -24,57 +24,172 @@
 #define _SDL_config_amiga_h
 
 #include "SDL_platform.h"
+#include <exec/types.h>
 
+#ifndef _HAVE_STDINT_H
 /* This is a set of defines to configure the SDL features */
 
-#define SDL_HAS_64BIT_TYPE	1
+#ifndef _INT8_T_DECLARED
+typedef signed char         			int8_t;
+#define _INT8_T_DECLARED
+#endif
+
+#ifndef _UINT8_T_DECLARED
+typedef unsigned char       			uint8_t;
+#define _UINT8_T_DECLARED
+#endif
+
+#ifndef _INT16_T_DECLARED
+typedef signed short        			int16_t;
+#define _INT16_T_DECLARED
+#endif
+
+#ifndef _UINT16_T_DECLARED
+typedef unsigned short      			uint16_t;
+#define _UINT16_T_DECLARED
+#endif
+
+#ifndef _INT32_T_DECLARED
+typedef signed int          			int32_t;
+#define _INT32_T_DECLARED
+#endif
+
+#ifndef _UINT32_T_DECLARED
+typedef unsigned int					uint32_t;
```

Comment:
A similar line in https://github.com/AmigaPorts/libSDL12/blob/324f6bc150eea7fbf25a0fb9a4d26f1ef6b9d984/include/SDL/SDL_config_amigaos.h#L53 caused a hard-to-diagnose link error in https://github.com/diasurgical/DevilutionX/pull/7996#issuecomment-2888154544